src/features/ajax/instrument/index.js

```diff
@@ -157,6 +157,8 @@ function subscribeToEvents (agentIdentifier, ee, handler, dt) {
 
     this.startTime = now()
 
+    this.body = data
```

Comment:
This is not needed & can be removed if this PR isn't affecting XHR (which it's not)

Reply:
it is... the body context is evaluated in the aggregator via parseGQL() call

Reply:
this is necessary for cases where the browser has been extended with a fetch polyfill, which uses xhr under the hood

Reply:
good catch thanks, yes it does

Reply:
added